### ISSUE TITLE
Add ADRF datasets

### ISSUE THREAD

**Issue by ernestogimeno** [Coleridge-Initiative/RCDatasets#240]
Use title and description as a reference. We need to look them up online and confirm the information and update it as needed.

ADRF_id | ADRF_title | ADRF_Provider | ADRF_description | ADRF_url
-- | -- | -- | -- | --
adrf-000015 | Chicago   Public Schools - School Progress Reports SY1516 | Chicago   Public Schools | 2015   school progress report ratings for all Chicago Public Schools. | https://data.cityofchicago.org/resource/fvrx-esxp
adrf-000061 | Illinois   Department of Human Services (IDHS) Point-In-Time Files: Time Invariant   Characteristics | Illinois   Department of Human Services (IDHS), Chapin Hall at the University of Chicago | The   point-in-time files are extracts from Chapin Hall’s Illinois Public   Assistance Research Database (ILPARD). This database is compiled from   sequential monthly point-in-time extracts from DHS. They cover participation   in a number of public assistance programs including Temporary Assistance to   Needy Families (TANF) and specific TANF initiatives (e.g., Work Pays),   Medicaid, KidCare, Food Stamps, Transitional Assistance, Aid to the Aged,   Blind, and Disabled (AABD), and Family Assistance. The data are mainly   cross-sectional but contain one historical component, an indicator of public   assistance receipt for the preceding 24 months. The most recent version of   the data is stored in the main tables and each table has an accompanying   change history. Because the spell files contain the time-varying history of   benefits receipt for individuals and cases, only the most recent version of   these tables is presented. As such, this data is presented as time-invariant,   with one record of each characteristic for a given individual or case, even   though some characteristics (like case addresses) obviously do vary over   time.               You will find the complete data in the class database under the tables:   idhs.assistance_case, idhs.case_geocode, idhs.case_social, idhs.member,   idhs.member_info, idhs.member_relation |  
adrf-000099 | LEHD   Origin-Destination Employment Statistics (LODES) Census Geography Crosswalk -   Kansas | US   Census Bureau | A   lookup table from Census Blocks to multiple other Census geographic entities,   as listed in the data dictionary. This is the geographic crosswalk file for   Kansas and is in the public schema of the "appliedda" database with   the name "lodes_census_geography_crosswalk_ks".                The primary key/identifier in the Geography Crosswalk is the 2010 Census   Tabulation Block Code (tabblk2010). This code does not change between   Decennial Censuses and can be used to link to the geocodes in the   Origin-Destination/Residential Area Characteristics/Workplace Area   Characteristics files as well as other datasets which include fields that   resolve to Census geography entities. All other geographic codes in the   crosswalk are “current” and represent the most recent definitions of the   legal, statistical, or administrative areas as integrated into LODES and   OnTheMap. | https://lehd.ces.census.gov/data/lodes/LODES7/ks/ks_xwalk.csv.gz

(3 datasets)

**Comment by JasonZhangzy1757** [comment]
**Hi @ernestogimeno,**

For the first dataset, all information is correct. 

For the second dataset, I can't find a specific dataset with its name, I guess it's a file list [here](http://www.dhs.state.il.us/page.aspx?module=1&item=27893), and somehow integrated. You could check the link and see if I think it right.

For the third dataset, all information is correct except description, which comes from nowhere. I guess someone must summarized it on his own.

**Comment by ernestogimeno** [comment]
For the second dataset, have you tried to look for "Public Assistance Research Database (ILPARD)" (from the description)?

**Comment by JasonZhangzy1757** [comment]
Yes, I have. But most related links are a PPT and an article from Chapin Hall, and I didn't see any link to the dataset or the introduction page to  the dataset. Could it be a private dataset owned by Chapin Hall?

**Comment by JasonZhangzy1757** [comment]
And I can find the introduction of the ILPARD dataset in this [article](https://www.degruyter.com/view/journals/bejeap/15/2/article-p589.xml?language=de), but no links towards it.

**Comment by ernestogimeno** [comment]
What do you think @ceteri ?

**Comment by ceteri** [comment]
@JasonZhangzy1757 @ernestogimeno:

I've been searching about `ILPARD` too. Neither of the local search services at U Chicago or Chapin Hall include any references to it, nor does a "link" search at Google with those two organizations.  

While I've found multiple papers that reference this as a distinct dataset:

 - https://www.researchconnections.org/childcare/resources/16511/pdf
 - https://www2.census.gov/ces/wp/2011/CES-WP-11-14.pdf
 - https://harris.uchicago.edu/files/errors_in_survey.pdf
 - https://www.degruyter.com/view/journals/bejeap/15/2/article-p589.xml?language=de
 - https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5115879/

It appears to be something associated with Bruce Meyer or Robert Goerge?

This is a good one to go back to Clayton and Julia to ask about, because there's no tangible evidence that it exists publically.
